shell/browser/browser_linux.cc

```diff
@@ -159,6 +160,13 @@ void Browser::ShowAboutPanel() {
     gtk_about_dialog_set_copyright(dialog, copyright.c_str());
   if (about_panel_options_.GetString("website", &website))
     gtk_about_dialog_set_website(dialog, website.c_str());
+  if (about_panel_options.Get("args", &args)) {
+    std::vector<const char*> cstrs{};
+    for (const auto& str : args) {
+      cstrs.push_back(str.c_str());
+    }
```

Comment:
This is literally just a sequence of pointers, C doesn't know the length. The idiom GTK+ uses here is for the sequence to be NULL-terminated:

```suggestion
    }
    cstrs.push_back(nullptr);
```

Otherwise the code as-is will just keep walking past the end of cstrs.data() and treat whatever random memory is there as char pointers